js/hubs/components/Release.js

```diff
@@ -263,9 +240,16 @@ const ReleaseComponent = ({ metadataSsr, releasePubkey, hubPubkey }) => {
                       e.stopPropagation()
                       downloadAs(
                         metadata.properties.files[0].uri,
-                        `${metadata.name
-                          .replace(/[^a-z0-9]/gi, '_')
-                          .toLowerCase()}___nina.mp3`
+                        releasePubkey,
+                        metadata.image,
```

Comment:
can `metadata` be passed as  a single prop and then the necessary values destructured within the function?